### PR TITLE
[Docs] Document FGAC user role assignment

### PR DESCRIPTION
**Pull request by mjseaman**
**What this PR does / why we need it**:
Adds documentation for user role assignment via API and using the role picker

**Which issue(s) this PR fixes**:
Fixes https://github.com/grafana/grafana-enterprise/issues/2174
FIxes https://github.com/grafana/grafana-enterprise/issues/2185
Fixes https://github.com/grafana/grafana-enterprise/issues/2170

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by eleijonmarck** [comment]
---
what I can help with:
* The specifics of the API endpoints (endpoint names, example requests and responses, and status codes)
* Roles and permissions for assigning roles to users in the FGAC reference files
* Sample YAML for granting roles to users using provisioning

There should be placeholder sections and text for each of those.
Would love to have this ready by Dec 1, so that “assigning roles to users” is fully unveiled for the main 8.3 release.

**Comment by vtorosyan** [comment]
Note that this PR also needs to be merged and is related https://github.com/grafana/grafana/pull/41398

**Comment by vtorosyan** [comment]
@IevaVasiljeva I've updated the following parts:
- Added documentation for the new API endpoints (list, assign and unassign user roles, list permissions assigned to users, bulk assignments)
- Updated fine-grained access control core documentation to clarify that direct assignments of roles to users now is possible
- Update the references with fixed roles to include new permissions, as well as the list of permissions.

Please let me know if there is anything else I can do to support with this.

**Comment by vtorosyan** [comment]
We need to merge changes from https://github.com/grafana/grafana/pull/42159/files which might be better/relevant.

File: docs/sources/enterprise/access-control/usage-scenarios.md [hunk 1]
```diff
@@ -132,6 +132,56 @@ Example response:
...
+
+There are two ways to grant roles directly to users: in the UI using the role picker, and using the API.
+
+### Grant roles to a user within a specific Organization using the role picker
+
+In order to assign roles to a user within a specific Organization using the role picker, you must have a user account with one of the following: 
+* The Admin builtin role.
+* The Server Admin role.
+* The fixed role `fixed:permissions:writer`, [assigned for the given Organization]({{< relref "roles/#scope-of-assignments" >}}).
+<!-- - A custom role with `[INSERT PERMISSIONS HERE]` permissions -->
```
Comment (osg-grafana) [inline docs/sources/enterprise/access-control/usage-scenarios.md:145]: @vtorosyan, can someone have a look at this line, as well as lines 162 and 181?

Comment (osg-grafana) [reply docs/sources/enterprise/access-control/usage-scenarios.md:145]: Alternatively, we can remove these embedded TODOs for now and track them as a new issue.

Comment (osg-grafana) [reply docs/sources/enterprise/access-control/usage-scenarios.md:145]: > We need to merge changes from https://github.com/grafana/grafana/pull/42159/files which might be better/relevant.

@IevaVasiljeva has restarted the build.

**Comment by osg-grafana** [comment]
I added a backport for this. Please let me know if this is incorrect.

**Review by osg-grafana — APPROVED** [review]
Approving, given those embedded TODOs are resolved or removed.

**Comment by osg-grafana** [comment]
Please rename `__index.md` to `_index.md`.

File: docs/sources/enterprise/access-control/manage-role-assignments/manage-user-role-assignments.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+You must also have the permissions granted by the roles that you want to assign or revoke.
+
+Steps:
+
+1. Navigate to the Users Configuration page by hovering over **Configuration** (the gear icon) in the left navigation menu and selecting **Users**.
+1. Click on the **Role** column in the row for the user whose role you would like to edit
+1. Select one or more roles that you would like to assign to that user.
+1. Click the **Apply** button to apply the selected roles to that user.
+
+![User role picker in Organization](/static/img/docs/enterprise/user_role_picker_global.png)
```
Comment (IevaVasiljeva) [inline docs/sources/enterprise/access-control/manage-role-assignments/manage-user-role-assignments.md:30]: PR for adding image assets: https://github.com/grafana/website/pull/6959

File: docs/sources/manage-users/server-admin/server-admin-manage-orgs.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ weight = 200
 
 # Manage organizations as a Server Admin
 
-This topic explains organization management tasks performed by Grafana Server Admins.
+This topic explains organization management tasks performed by Grafana Server Adhttp_api/access_control.mdmins.
```
Comment (sakjur) [inline docs/sources/manage-users/server-admin/server-admin-manage-orgs.md:8]: 🙂

```suggestion
This topic explains organization management tasks performed by Grafana Server Admins.
```

?

Comment (IevaVasiljeva) [reply docs/sources/manage-users/server-admin/server-admin-manage-orgs.md]: Yup :) Thanks for spotting!

**Comment by vtorosyan** [comment]
@IevaVasiljeva this is really good work, thanks for the contributions. I have made few small updates and the only improvement which comes to mind is perhaps reordering the page structure so that we have Roles menu item and the rest under it, but we can discuss this in future milestones and see if it would make sense or no, nothing to do now.

**Review by vtorosyan — APPROVED** [review]

**Pull request merged**